package_installs.R

```diff
@@ -48,3 +48,4 @@ install.packages("prophet")
 # handled in this image's Dockerfile.
 install.packages("fftw")
 install.packages("seewave")
+install.packages("Rdimtools")
```

Comment:
We automatically install all packages from R CRAN in the `docker-rcran` image (base image for `docker-rstats`:
https://github.com/Kaggle/docker-rcran

However, the Rdimtools failed to install. We only fail the build if the package is in our important package list:
https://github.com/Kaggle/docker-rcran/blob/master/test_build.R

The reason for this is that we install 10k+ packages from R CRAN maybe only 80% of them install successfully.

You can pull the `docker-rcran` image and try to install Rdimtools manually, I bet it fails because of the missing `libmpfr-dev` dependency you have added above. Please, add the missing dependency to the `docker-rcran` image and send us a PR. You can `cc/ @rosbo` me on the PR and I will review it promptly. Thank you